dbt_project.yml

```diff
@@ -0,0 +1,36 @@
+name: 'mozdbt'
+version: '1.0.0'
+config-version: 2
+
+# This setting configures which "profile" dbt uses for this project.
+profile: 'mozdbt'
+
+# These configurations specify where dbt should look for different types of files.
+# The `source-paths` config, for example, states that models in this project can be
+# found in the "models/" directory. You probably won't need to change these!
+model-paths: ["models"]
+analysis-paths: ["analysis"]
+test-paths: ["tests"]
+data-paths: ["data"]
+macro-paths: ["macros"]
+snapshot-paths: ["snapshots"]
+
+target-path: "target"  # directory which will store compiled SQL files
+clean-targets:         # directories to be removed by `dbt clean`
+  - "target"
+  - "dbt_packages"
+
+models:
+  mozdbt:
+    moz-fx-data-shared-prod:
+      +on_schema_change: "append_new_columns"
+      +materialized: incremental
+      business_models:
+        +persist_docs:
```

Comment:
Nice! This is a useful config to turn on. Should we turn this on for all models, instead of at the directory level?

Reply:
I figured we might not need docs for `intermediate` models, so for now I only enabled it for the `business_models`

Reply:
I like that, it makes the models less likely to be used by random people that find them in BQ.